educe/stac/fusion.py

```diff
@@ -197,3 +201,47 @@ def speaker(self):
 # pylint: disable=invalid-name
 FakeRootEDU = _FakeRootEDU()
 # pylint: enable=invalid-name
+
+
+def fuse_edus(discourse_doc, unit_doc, postags):
+    """Return a copy of the discourse level doc, merging info
+    from both the discourse and units stage.
+
+    Any unit annotations which happen to be EDUs will be converted
+    to higher level EDUs
```

Comment:
EDUs from both levels seem to coexist inside of the new EDU.
If so, this last sentence is slightly confusing.